lib/rails_erd/domain/relationship.rb

```diff
@@ -63,7 +63,8 @@ def association_target(association)
 
       def initialize(domain, associations) # @private :nodoc:
         @domain = domain
-        @reverse_associations, @forward_associations = *unless any_habtm?(associations)
+        @reverse_associations, @forward_associations =
+        unless any_habtm?(associations)
```

Comment:
I like this better, but it's still a little hard to read. How about a new private function for this unless/else block?  ..additionally, I'm personally in favor of not using `else` with an `unless` whenever possible; could you flip the logic?